sklearn/naive_bayes.py

```diff
@@ -753,6 +753,9 @@ class ComplementNB(BaseDiscreteNB):
     class_prior : array-like, size (n_classes,), optional (default=None)
         Prior probabilities of the classes. Not used.
 
+    norm : boolean, optional (default=False)
+        Whether or not to perform second normalization of weights.
```

Comment:
I think this deserves a further comment to say that the default behaviour replicates that of Mahout (what does Weka do, I wonder), as opposed to the paper.

Reply:
Done. Mahout doesn't do [the second normalization by default](https://github.com/Spirit-Dongdong/weka-learning/blob/master/src/main/java/weka/classifiers/bayes/ComplementNaiveBayes.java#L104) either.